### DISCUSSION TITLE
Import Stock Option Feature?

### DISCUSSION THREAD

**Discussion by chart-analyst** [ghostfolio/ghostfolio#2333] [Ideas]
Starting with a flex query from Interactive Brokers, I began my first csv import into Ghostfolio.  I basically just created a bunch of test import files with a few lines, trying different things.  Most failed, but it wasn't difficult to identify the problem most times (I did find the ok.csv example file, which was helpful).

My primary observation is Ghostfolio does not like stock options.  I searched for a discussion on stock options, found nothing.  So I tried it.  I used a multiplier of 100 on all option quantities to emulate the correct absolute P&L.  But Ghostfolio does not like symbol names larger than 4 characters, so I had to scale back to the underlying symbol.  When comments are available, I will put a note in identifying the trade as an option.

For this particular account, this is a big issue - around 2/3 of my trades in this account are options. I have other accounts where it is not as big a deal, but I trade options in all of my accounts.

So the obvious question is, are there plans to add options to Ghostfolio?  Not a trivial task, I know.  But worth discussing.  If there are other posts that I missed, please point me in the right direction.  Thanks.

**Comment by dtslvr** [comment]
I think your idea is related to #969.
